### PR TITLE
docs: Made some edits

### PR DESCRIPTION
**Pull request by LamiWilliams**
closes #ISSUE-NO

#### What does this PR do?
It contains some restructured sentences and corrected typos

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Raise an [issue to join the EddieHub GitHub organisation](https://github.com/Edd
 
 ### Some vague idea of how we could turn this into a support channel.
 
-Imagine the scenario where you are helping a stranger (or a new friend), a friend or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported, a Node.js article that isn't supported in Node v14 and that sort of thing...
+Imagine the scenario where you are helping a stranger, a friend or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported or a Node.js article that isn't supported in Node v14. That sort of thing...
 
-You volunteer to help that person, you pair on it, it's fixed and they're super happy.
+You volunteer to help that person and pair on it. You fix it. They're super happy.
```
Comment (vinzvinci) [inline README.md:11]: I think this one's better?:
`You volunteer to help that person and pair on it. You fix it and that made them super happy.`

Comment (LamiWilliams) [reply README.md:11]: >You volunteer to help that person and pair on it. You fix it and that made them super happy.

This looks good. But, how about changing "made" to "make" to maintain the present tense being used?

Comment (vinzvinci) [reply README.md:11]: Yeah better use "make"

make = present
made = past

Comment (LamiWilliams) [reply README.md:11]: Making the change now

**Review by vinzvinci — DISMISSED** [review]
LGTM 👍

**Review by schmelto — DISMISSED** [review]

**Review by chaitanyasjoshi — DISMISSED** [review]

**Review by bishalr0y — APPROVED** [review]

**Review by bishalr0y — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -6,11 +6,11 @@ Raise an [issue to join the EddieHub GitHub organisation](https://github.com/Edd
 
 ### Some vague idea of how we could turn this into a support channel.
 
-Imagine the scenario where you are helping a stranger (or a new friend), a friend or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported, a Node.js article that isn't supported in Node v14 and that sort of thing...
+Imagine the scenario where you are helping a stranger, a friend or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported or a Node.js article that isn't supported in Node v14. That sort of thing...
```
Comment (adityaraute) [inline README.md:9]:
```suggestion
Imagine the scenario where you are helping a stranger, a friend, or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported or a Node.js article that isn't supported in Node v14. That sort of thing...
```


Oxford commas are necessary

File: README.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ Raise an [issue to join the EddieHub GitHub organisation](https://github.com/Edd
...
 
-You volunteer to help that person, you pair on it, it's fixed and they're super happy.
+You volunteer to help that person and pair on it. You fix it and that makes them super happy.
```
Comment (adityaraute) [inline README.md:11]:
```suggestion
You volunteer to help that person and pair on it. You fix it and they get super happy.
```

Might be economical?

File: README.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ Please check out the [issue queue](https://github.com/EddieJaoudeCommunity/suppo
 - **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be: [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook) and of course [EddieHub](https://github.com/EddieHubCommunity).
 Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-organizations).
 - **How do I join the Github Organization?** Raise an issue on this repository asking to join ([invite me](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization)).
-- **How do I set the Organization to Public?** You can make the organization public by: 
+- **How do I set the Organization to Public?** Here's how to make the organization public:
```
Comment (adityaraute) [inline README.md:51]: Is this one really necessary? Rethink.

File: README.md [hunk 5]
```diff
@@ -66,7 +66,7 @@ Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-man
...
-- **How do I get started with open source?** You can make your first pull request in this [repository](https://github.com/EddieJaoudeCommunity/hacktoberfest-practice). Follow the instructions in the README, and don't forget contributing to open source is not just about code, it's about collaboration, communication, and adding value. Don't be afraid to open an issue with a question :smiley:. If you have any trouble you can ask questions in our Discord, we have a channel for `first-timers` and `opensource`.
+- **How do I get started with open source?** You can make your first pull request in this [repository](https://github.com/EddieJaoudeCommunity/hacktoberfest-practice). Follow the instructions in the README, and don't forget contributing to open source is not just about code, it's about collaboration, communication, and adding value. Don't be afraid to open an issue with a question :smiley:. If you have any trouble, you can ask questions in our Discord. We have a channel for `first-timers` and `opensource`.
```
Comment (adityaraute) [inline README.md:69]:
```suggestion
- **How do I get started with open source?** You can make your first pull request in this [repository](https://github.com/EddieJaoudeCommunity/hacktoberfest-practice). Follow the instructions in the README, and don't forget – contributing to open source is not just about code – it's about collaboration, communication, and adding value. Don't be afraid to open an issue with a question :smiley:. If you have any trouble, you can ask questions in our Discord. We have a channel for `first-timers` and `opensource`.
```

How about this structure instead?

**Review by adityaraute — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (LamiWilliams) [reply README.md:51]: @adityaraute The later sounds more direct. The listed items beneath would be able to start with needing to add "-ing"

Comment (krishdevdb) [inline README.md:51]:
```suggestion
- **How do I set the Organization to Public?** Here's how to publicize your organization membership:
```

**Review by krishdevdb — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (krishdevdb) [reply README.md:51]: Maybe this will be better 🆙

Comment (schmelto) [reply README.md:51]: Why not deleting the whole second sentence?
It's really redundant information why writing it two times :)
```suggestion
- **How do I set the Organization to Public?**
```

Comment (krishdevdb) [reply README.md:51]: I like your suggestion better, but I think that sentence must be there for some reason?

Comment (schmelto) [reply README.md:51]: I could not think of any reason why we should keep this sentence 🙈

Comment (adityaraute) [reply README.md:51]: agree with @schmelto

**Review by schmelto — APPROVED** [review]

**Pull request merged**